### ISSUE TITLE
Void Fiend Text Error

### ISSUE THREAD

**Issue by JamesTripleQ** [xoxfaby/BetterUI#115]
**Describe the bug**
The description for Void Fiend's Special is always it's corrupted variant both on the lobby menu and when hovering over the skill in-game.

**Does the error occur when only using BetterUI and no other mods?**
This error seems to be entirely tied to BetterUI and more specifically the AdvancedIcon component since disabling that fixes the issue.

**BetterUI Version** 
2.5.10

**Comment by xoxfaby** [comment]
Can you confirm what happens in VANILLA?

I added extra skill tokens because in vanilla, both forms bad exactly the same ones. 

Idk if this has changed

**Comment by JamesTripleQ** [comment]
Vanilla
![image](https://user-images.githubusercontent.com/90627380/163445233-d5aeecfd-65f8-4b30-a50a-eb98323a7672.png)
With BetterUI
![image](https://user-images.githubusercontent.com/90627380/163446753-99f9031b-bf14-46f9-997a-47d8c357c17c.png)

**Comment by xoxfaby** [comment]
I see, this is strange, what about in-game?

**Comment by JamesTripleQ** [comment]
Same thing in game... Vanilla has the normal one and BetterUI always has the corrupted version

**Comment by JamesTripleQ** [comment]
Sorry I couldn't get the screenshots earlier

Vanilla
![image](https://user-images.githubusercontent.com/90627380/163571039-78a9baa0-3233-4f8b-8cf4-44b9c4e856a1.png)

With BetterUI
![image](https://user-images.githubusercontent.com/90627380/163571209-e50e9503-110e-4279-bfff-42466c7f3375.png)

**Comment by JamesTripleQ** [comment]
I'm no modder for this game so I'm probably wrong but I think that the error lies in line 73 of `AdvancedIcons.cs`, it mentions "CrushCorruption" but that is the name of the un-corrupted skill... I think you might need to change that to the corrupted version.

**Comment by JamesTripleQ** [comment]
I also just noticed that the Corrupt Suppress is also wrong and displays the normal Suppress
![image](https://user-images.githubusercontent.com/90627380/164302572-11f6fc72-9d07-4eaf-b749-eba3c7f812ba.png)

**Comment by xoxfaby** [comment]
Pretty sure I fixed this, if I haven't please reopen this issue

**Comment by JamesTripleQ** [comment]
Hi, sorry this still seems to happen even on the latest version of BetterUI.

**Comment by xoxfaby** [comment]
I tested this today and it was working correctly, in Vanilla there is only 1 description and with BetterUI it shows the correct ones.

**Comment by xoxfaby** [comment]
What language do you play in?

**Comment by xoxfaby** [comment]
Please also provide a log, until then the issue remains closed again

**Comment by JamesTripleQ** [comment]
My PC's language is Italian but my game is in English.

The following three screenshots DO NOT have BetterUI:

![image](https://user-images.githubusercontent.com/90627380/200193905-e8e8e79b-666c-4498-968e-3621ab83319c.png)
![image](https://user-images.githubusercontent.com/90627380/200193938-c480df4a-2a12-4c6f-8899-0c5b4e3bd2ae.png)
![image](https://user-images.githubusercontent.com/90627380/200193942-8580f1ab-9e7a-41c6-b861-9f97bfc6fe5a.png)

These next few screenshots are WITH BetterUI:

![image](https://user-images.githubusercontent.com/90627380/200193973-d4e7bc24-3f2d-4ad3-9b88-675586b94abd.png)
![image](https://user-images.githubusercontent.com/90627380/200193983-0461ea98-1e49-4239-a027-ca0b4967d00a.png)
![image](https://user-images.githubusercontent.com/90627380/200193994-a7a1b555-d0cc-4800-9663-304af592819e.png)

There are a few things to note:
1) in all images of BetterUI the text for Suppress and Corrupted Suppress (Still called Suppress in game) is swapped: in the character select menu both the skill name and "[Corruption Upgrade]" text are swapped, and it's like that in the Tab menu descriptions too.
2) in the 3rd NON BetterUI image the Corrupted Suppress text hasn't changed from the regular Suppress text, this seems to be something the devs never implemented

**Comment by JamesTripleQ** [comment]
As for the log here it is however it also has other mods enabled since, I don't know why, but disabling them seems to corrupt my save
[LogOutput.log](https://github.com/xoxfaby/BetterUI/files/9946823/LogOutput.log)

**Comment by xoxfaby** [comment]
Yeah, so the Vanilla behaviour doesn't have any text for the other skills because this is a feature of BetterUI, the risk of rain 2 dev forgot I guess and BetterUI specifically adds extra text for those skills. As for BetterUI, it seems that Suppress is backwards yeah, now with this more detailed info I was able to confirm what you're seeing because I misunderstood before and thought you meant that all skills were backwards. Thank you so much for your bug report and detailed info. 

I will close this issue when I've deployed the patch that fixes this.
